bookwyrm/urls.py

```diff
@@ -276,6 +276,7 @@
     # users
     re_path(rf"{USER_PATH}\.json$", views.User.as_view()),
     re_path(rf"{USER_PATH}/?$", views.User.as_view(), name="user-feed"),
+    re_path(rf"@(?P<username>{regex.USERNAME})$", views.user_redirect),
```

Comment:
This just needs a `^` at the beginning of the string, otherwise it will match any url that ends in `@<username>`